package.xml

```diff
@@ -21,10 +21,12 @@
   <url type="bugtracker">https://github.com/ros2/kdl_parser/issues</url>
 
   <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>eigen3_cmake_module</buildtool_depend>
 
-  <depend>orocos_kdl_vendor</depend>
+  <depend>liborocos-kdl</depend>
```

Comment:
Same need for liborocos-kdl-dev build_depend and build_export_depend as https://github.com/ros2/geometry2/pull/826#discussion_r2322354234